frontendservice/src/views/ShopOwnerHome.vue

```diff
@@ -1,79 +1,170 @@
-/* Copyright 2020 Google LLC Licensed under the Apache License, Version 2.0 (the
-"License"); you may not use this file except in compliance with the License. You
-may obtain a copy of the License at https://www.apache.org/licenses/LICENSE-2.0
-Unless required by applicable law or agreed to in writing, software distributed
-under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
-CONDITIONS OF ANY KIND, either express or implied. See the License for the
-specific language governing permissions and limitations under the License. */
+/*
+Copyright 2020 Google LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
 
 <template>
-  <div class="home">
-    <h1 v-if="isLoggedIn">Welcome! {{ name }}</h1>
-    <p>
-      This page will be Home page for the shop-owners. 
-    </p>
-    <p>
-      ToDo: Implement ShopOwnerHome page features including NavBar, Search Bar, MyShops and AddShop.
-    </p> 
+  <div>
     <div v-if="isVerified">
-      <router-link to="/create-shop">Create new shop</router-link>
-    </div>
-    <div v-if="isVerified">
-      <router-link to="/my-shops">My Shops</router-link>
+      <shopOwnerNavbar />
+      <h1 id="heading" v-if="isVerified">My Shops</h1>
+      <input type="search" v-model="searchResult" placeholder="search shops" id="searchbar" />
```

Comment:
Nit. For unification purposes: please refer to Bavanya's search bar code (PR #40 ), so that the search bars on both the pages look similar.

Reply:
Maybe we can have a call for unification later to decide what's best.